src/test/rustdoc/sized_trait.rs

```diff
@@ -0,0 +1,15 @@
+#![crate_name = "foo"]
+
+// @has foo/struct.Bar.html
+// @has - '//h3[@id="impl-Sized"]' 'impl Sized for Bar'
+pub struct Bar {
+    a: u16,
+}
+
+// @has foo/struct.Foo.html
+// @has - '//h3[@id="impl-Sized"]' 'impl<T> Sized for Foo<T>'
```

Comment:
Shouldn't this be `impl<T: Sized> Sized for Foo<T>`?

Reply:
The compiler assumes by default that the generics are `Sized` iiuc.

Reply:
Ah, yeah, in that case there should be a testcase for `struct Foo<T: ?Sized>(T);` too.

Reply:
Good idea. I'll add it when I'll make the update.